docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,614 @@
+  instructions on using Terraform modules to deploy an HPCC Systems instance
+  specifically on the Azure Cloud.</para>
+
+  <para>These modules were developed by the HPCC Systems platform team for
+  general open-source community usage. You may require specific customizations
+  for your particular needs. For example, your organization may require
+  opinionated modules for production systems. You can develop your own
+  customized modules, per your requirements and utilize them in the same
+  manner outlined here. </para>
+
+  <sect1 id="GS_DeployingHPCC_w_Terraform" role="nobrk">
+    <title>Getting Started With Terraform </title>
+
+    <para>This section details deploying the containerized HPCC Systems
+    platform onto Azure using Terraform. Using the open source and additional
+    modules from the HPCC Systems Terraform open-source repository. No
+    previous knowledge of Terraform, Kubernetes, or Azure is
+    required.  </para>
+
+    <para>The steps to deploy an HPCC Systems instance using our provided
+    Terraform modules are detailed in the subsequent sections. A short summary
+    of these steps is as follows. </para>
+
+    <orderedlist>
+      <listitem>
+        <para>Clone the HPCC Systems Terraform module repository</para>
+      </listitem>
+
+      <listitem>
+        <para>Modify the configuration files (admin.tfvars) </para>
+      </listitem>
+
+      <listitem>
+        <para>Initialize the modules</para>
+      </listitem>
+
+      <listitem>
+        <para>Apply the initialized modules</para>
+      </listitem>
+    </orderedlist>
+
+    <para>The strength of using Terraform modules to deploy your system, you
+    only need to set them up once. After they are in place and configured, you
+    can reuse them to stand up an identical instance of your system. You can
+    do so by initializing and then applying them. </para>
+
+    <sect2 id="TerraformRequirements">
+      <title>Requirements</title>
+
+      <para>What you will need in order to deploy an HPCC Systems instance
+      with Terraform:</para>
+
+      <itemizedlist>
+        <listitem>
+          <para>A Linux, MacOS, or Windows OS computer system.</para>
+        </listitem>
+
+        <listitem>
+          <para>A browser. Such as Chrome or Firefox.</para>
+        </listitem>
+
+        <listitem>
+          <para>Git and a Github account that you can access and clone the
+          repository.</para>
+        </listitem>
+
+        <listitem>
+          <para>An Azure account with sufficient permissions, rights, credits,
+          and credentials. To obtain one, go to www.azure.com or talk to your
+          manager if you believe that your employer might have a corporate
+          account.</para>
+        </listitem>
+
+        <listitem>
+          <para>A code editor of your choice. There are a few editors
+          integrated with Azure such as code, vi the Visual Editor, Nano, or
```

Comment:
instead of "code" say VS Code - there are probably rules about mentioning specific 3rd party apps